.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: ruby
+
+rvm:
+- 2.6.6
+
+addons:
+  postgresql: '13'
+  apt:
+    packages:
+      - postgresql-13
+env:
+  global:
+    - PGPORT=5433
+
+services:
+  - postgresql
+
+# cache:
```

Comment:
Did you intend for all this to be commented out?

Reply:
same as below. was unsure if i actually needed this stuff...